DEVELOPMENT.md

```diff
@@ -8,9 +8,9 @@ All development tasks consist of watching the demo bundle and the test bundle.
 Run the application with watched rebuilds:
```

Comment:
@coopy since this application actually does provide some additional functionality in the app that requires documentation, I was planning to replace it with something close to that in the builder-init version of the new archetype: https://github.com/FormidableLabs/builder-react-app/blob/converter-react-archetype-init/init/DEVELOPMENT.md

Can you give feedback on that file?